pkgs/development/libraries/noise-suppression-for-voice/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "noise-supression-for-voice";
+  version = "2020-10-12";
+
+  src = fetchgit {
```

Comment:
Prefer `fetchFromGitHub` here, for lightweight tarballs from github's API: https://nixos.org/manual/nixpkgs/stable/#sec-sources